app/js/helpers/apiHelper.js

```diff
@@ -1,5 +1,8 @@
 import fetchPolyfill from 'whatwg-fetch';
 
+const contextPath = location.href.split('/')[3];
+const BASE_URL = `/${contextPath}/ws/rest/v1/`;
```

Comment:
The url showing on the browser will determine what the contextPath will be. For example if the user is using openmrs on http://localhost:8080/openmrs/. Then the contextPath will be `openmrs` and the `BASE_URL` for the API endpoints will be `/openmrs/ws/rest/v1/